src/components/Calculator.js

```diff
@@ -1,44 +1,60 @@
-            <div className="input">0</div>
+            <div className="input">{next || total || '0'}</div>
           </div>
         </div>
         <div className="row">
-          <div className="operation">AC</div>
-          <div className="operation">+/-</div>
-          <div className="operation">%</div>
-          <div className="operation right">÷</div>
+          <button type="button" onClick={this.onClickHandler} className="operation">AC</button>
+          <button type="button" onClick={this.onClickHandler} className="operation">+/-</button>
+          <button type="button" onClick={this.onClickHandler} className="operation">%</button>
+          <button type="button" onClick={this.onClickHandler} className="operation right">÷</button>
         </div>
         <div className="row">
-          <div className="number">7</div>
-          <div className="number">8</div>
-          <div className="number">9</div>
-          <div className="operation right">X</div>
+          <button type="button" onClick={this.onClickHandler} className="number">7</button>
+          <button type="button" onClick={this.onClickHandler} className="number">8</button>
+          <button type="button" onClick={this.onClickHandler} className="number">9</button>
+          <button type="button" onClick={this.onClickHandler} className="operation right">x</button>
         </div>
         <div className="row">
-          <div className="number">4</div>
-          <div className="number">5</div>
-          <div className="number">6</div>
-          <div className="operation right">-</div>
+          <button type="button" onClick={this.onClickHandler} className="number">4</button>
+          <button type="button" onClick={this.onClickHandler} className="number">5</button>
+          <button type="button" onClick={this.onClickHandler} className="number">6</button>
+          <button type="button" onClick={this.onClickHandler} className="operation right">-</button>
         </div>
         <div className="row">
-          <div className="number">1</div>
-          <div className="number">2</div>
-          <div className="number">3</div>
-          <div className="operation right">+</div>
+          <button type="button" onClick={this.onClickHandler} className="number">1</button>
+          <button type="button" onClick={this.onClickHandler} className="number">2</button>
+          <button type="button" onClick={this.onClickHandler} className="number">3</button>
+          <button type="button" onClick={this.onClickHandler} className="operation right">+</button>
         </div>
         <div className="row bottom">
-          <div className="number zero">0</div>
-          <div className="operation">.</div>
-          <div className="operation right">=</div>
+          <button type="button" onClick={this.onClickHandler} className="number zero">0</button>
+          <button type="button" onClick={this.onClickHandler} className="operation">.</button>
```

Comment:
**Changes Required**
- [ ] There is too much repetition of the `button` here making our code not follow the `Don't Repeat Yourself` principle. Let us try to refactor this into a `Button Component` and  pass the name and onClick event as `props`.
- [ ] Let us try to store all the button names in an `array`. In that way, we can then use `array.map` to dynamically add all the buttons into the `Virtual DOM`.